lib/test_tools.py

```diff
@@ -59,9 +63,16 @@ def MakeYuvFileWithOneBlankFrame(name):
   """Make an YUV file with one black frame."""
   return MakeYuvFileWithBlankFrames(name, 1)
 
+def EmptyWorkDirectory():
+  dirname = encoder_configuration.conf.workdir()
+  if dirname[0:4] != '/tmp':
+    raise encoder.Error('Workdir is %s, not a tempfile' % dirname)
+  shutil.rmtree(dirname)
+  os.mkdir(dirname)
+
 def FinishWorkDir(dirname):
   # Verification of validity
-  if os.environ['CODEC_WORKDIR'] != dirname:
+  if encoder_configuration.conf.sysdir() != dirname:
     raise encoder.Error('Dirname was wrong in FinishWorkDir')
```

Comment:
Improve this error message: raise encoder.Error('Got dirname %s but expected %s.' % (dirname, ...))